core/src/models/Schedule.ts

```diff
@@ -314,7 +314,7 @@ export class Schedule extends LoggedModel<Schedule> {
   @BeforeSave
   static async ensureSourceMapping(instance: Schedule) {
     const source = await Source.findById(instance.sourceId);
-    if (!source.scheduleAvailable()) {
+    if (!(await source.scheduleAvailable())) {
```

Comment:
^ Here's the problem

Reply:
Should there be a test for this?

Reply:
Curiously, there already is! https://github.com/grouparoo/grouparoo/blob/d1947cd4e96993a3a37b9c2e54890d980ab32787/core/__tests__/models/schedule.ts#L294-L355

Reply:
ooh! Actually, this method really doesn't do much any more - I think we can remove it